services/QuillLMS/lib/tasks/convert_activity_session_timetracking_data.rake

```diff
@@ -0,0 +1,12 @@
+namespace :activity_session_timetracking_data do
+  desc 'updates data that had formerly been saved as json in hstore to be parsable in jsonb'
+  task :convert => :environment do
+    ActivitySession.where("activity_sessions.data->'time_tracking' IS NOT NULL").each do |as|
+      if as.data['time_tracking'].instance_of? String
+        time_tracking = JSON.parse(as.data['time_tracking'])
+        as.data = { time_tracking: time_tracking }
```

Comment:
If I remember correctly, only Comprehension activities should have `time_tracking` as a key in `data`, and they have nothing else in `data`, so it's safe to just overwrite the values in `data` for these records completely.  Just want to confirm that that's true just in case what we'd rather do is something like `as.data['time_tracking'] = time_tracking` or something.